spring-rabbit/src/main/java/org/springframework/amqp/rabbit/config/ListenerContainerFactoryBean.java

```diff
@@ -366,8 +368,17 @@ public void setReceiveTimeout(long receiveTimeout) {
 		this.receiveTimeout = receiveTimeout;
 	}
 
+	public void setBatchSize(int batchSize) {
+		this.batchSize = batchSize;
+	}
+
+	@Deprecated
 	public void setTxSize(int txSize) {
-		this.txSize = txSize;
+		setBatchSize(txSize);
+	}
+
+	protected void setConsumerBatchEnabled(boolean consumerBatchEnabled) {
```

Comment:
Why is this `protected` ? Are you going to detect this option internally by the `@RabbitListener(Handler)` method signature?

Reply:
stupid IDE (or IDE driver)